.gitignore

```diff
@@ -18,4 +18,8 @@ ApacheJMeterTemporaryRootCA.crt
 ApacheJMeterTemporaryRootCA.usr
 player/params/
 player/test.json
-player/test.json.pretty
\ No newline at end of file
+player/test.json.pretty
+
+# IDEA
```

Comment:
I prefer to store  some IDEA's project files in separate folder, it solves that issue with charm,  no need to exclude IDEA's files from git